source/blog/2015-03-01-cappuccino-news-2015-03-01.markdown

```diff
@@ -0,0 +1,74 @@
+---
+title: Cappuccino News 2015 March 1st
+author: Alexandre Wilhelm
+author_email: alexandre.wilhelmfr@gmail.com
+date: '2015-03-01'
+tags:
+- Cappuccino
+- Cucapp
+- Cucumber
+- Objective-J
+- update
+- OJTest
+---
+
+Here's a quick update on what's new with Cappuccino for the last month.
+
+### Cappuccino
+
+##### News
+
+- The graphical `CPDatePicker` now disables the dates outside the `minDate` and `maxDate`. ([daboe01](https://github.com/daboe01) - [PR](https://github.com/cappuccino/cappuccino/pull/2297))
+
+- `Cappuccino` now draws by default in high DPI in a `canvas2D`. You can disable this feature using the `CPView` class method `setAllowsHighDPIDrawing:`. ([Dogild](https://github.com/Dogild) - [PR](https://github.com/cappuccino/cappuccino/pull/2315))
```

Comment:
You can disable this feature globally using the method `CPView +setAllowsHighDPIDrawing:`

Wrong pull request, it's https://github.com/cappuccino/cappuccino/pull/2215